fastly/ngwaf/v1/events/api_list.go

```diff
@@ -0,0 +1,75 @@
+package events
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/fastly/go-fastly/v10/fastly"
+)
+
+// GetInput specifies the information needed for the Get() function to perform
+// the operation.
+type ListInput struct {
+	// Context, if supplied, will be used as the Request's context.
+	Context *context.Context `url:"-"`
```

Comment:
It doesn't appear that any of the tags in this structure are needed, as the structure is never passed to any serializer.